off/off.go

```diff
@@ -0,0 +1,202 @@
+// Package off provides classes that write OFF files
+// OFF files store TES pulses projected into a linear basis
+// OFF files have a JSON header followed by a single newline
+// after the header records are written sequentially in little endian format
+// bytes		type			meaning
+// 0-3      int32     recordSamples (could be calculated from nearest neighbor pulses in princple)
+// 4-7      int32     recordPreSamples (could be calculated from nearest neighbor pulses in princple)
+// 8-15     int64     framecount
+// 16-23    int64     timestamp from time.Time.UnixNano()
+// 24-27    float32   pretriggerMean (from raw data, not from modeled pulse, really shouldn't be neccesary, just in case for now!)
+// 28-31    float32   residualStdDev (in raw data space, not Mahalanobis distance)
```

Comment:
This residual is simpler, but I dream someday of computing Mahalanobis distance. Will the header contain a version number to permit changes in this particular OFF format? And anyway, aren't you going to wish we had used [gogetdata](https://github.com/joefowler/gogetdata) pretty soon?